micrometer-test/src/main/java/io/micrometer/core/tck/DefaultLongTaskTimerTest.java

```diff
@@ -35,8 +35,12 @@
 import static io.micrometer.core.instrument.MockClock.clock;
 import static org.assertj.core.api.Assertions.assertThat;
 
+// Test class is public and in main code... by mistake?
+//CHECKSTYLE:OFF
 public class DefaultLongTaskTimerTest {
```

Comment:
We should probably consider copying this to test code separately from this pull request and marking this as deprecated - removing it would be breaking public API, unfortunately. I don't think these tests are being run right now because this is in main code and I don't see this class referenced from anywhere in our code base.